### PR TITLE
Update update-and-cleanup.md

### PR DESCRIPTION
**Pull request by jpduyx**
spotify dockergc is UNMAINTAINED, they advice to consider using docker system prune command instead.

"This repository has been archived by the owner on Feb 2, 2021. It is now read-only."

https://github.com/spotify/docker-gc

# Description

Fixes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [ ] If necessary I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/content/config/advanced/maintenance/update-and-cleanup.md [hunk 1]
```diff
@@ -23,19 +23,6 @@ For more details, see the [manual](https://containrrr.github.io/watchtower/)
 
 ## Automatic Cleanup
 
-When you are pulling new images in automatically, it would be nice to have them cleaned up as well. There is also a docker image for this: [`spotify/docker-gc`](https://hub.docker.com/r/spotify/docker-gc/).
+For cleanup you can use the [`--cleanup`](https://containrrr.github.io/watchtower/arguments/#cleanup) option provided by `containrrr/watchtower`.
```
Comment (polarathene) [inline docs/content/config/advanced/maintenance/update-and-cleanup.md:26]: Might as well keep some context about the intent, along with a compose example:

````suggestion
The service [`containrrr/watchtower`][watchtower] can monitor your images and automatically update them. Enable the [`--cleanup` option][watchtower-cleanup] to remove orphaned images from storage that are no longer used.

[watchtower]: https://containrrr.dev/watchtower/
[watchtower-cleanup]: https://containrrr.github.io/watchtower/arguments/#cleanup

```yaml
services:
  watchtower:
    image: containrrr/watchtower
    environment:
      - WATCHTOWER_CLEANUP=true
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

!!! tip "Only update specific containers"

    The default `watchtower` service will poll every 24 hours for all images to update, not just within your `compose.yaml`.

    View their docs to provide a list of containers names and other settings if you'd like to restrict that behaviour.
````

File: docs/content/config/advanced/maintenance/update-and-cleanup.md [hunk 2]
```diff
@@ -23,19 +23,6 @@ For more details, see the [manual](https://containrrr.github.io/watchtower/)
...
-    restart: always
-    image: spotify/docker-gc:latest
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
-```
-
-For more details, see the [manual](https://github.com/spotify/docker-gc/blob/master/README.md)
-
-Or you can just use the [`--cleanup`](https://containrrr.github.io/watchtower/arguments/#cleanup) option provided by `containrrr/watchtower`.
+Or consider using ['docker prune command'](https://docs.docker.com/engine/reference/commandline/system_prune/)
```
Comment (polarathene) [inline docs/content/config/advanced/maintenance/update-and-cleanup.md:28]: A little more context. `docker image prune` is probably sufficient here than `docker system prune`. Clarified that this isn't automatic / scheduled, but manually invoked.

```suggestion
!!! tip "Manual cleanup"

    While `watchtower` also supports running once with `docker run` or `compose.yaml` with the `--run-once` option, you can also directly invoke cleanup with [`docker image prune --all`][docker-docs-prune-image] or the more thorough [`docker system prune --all`][docker-docs-prune-system] (_also removes unused containers, networks, build cache_). Avoid the `--all` option if you only want to cleanup "dangling" content.

[docker-docs-prune-image]: https://docs.docker.com/engine/reference/commandline/image_prune/
[docker-docs-prune-system]: https://docs.docker.com/engine/reference/commandline/system_prune/
```

**Review by polarathene — CHANGES_REQUESTED** [review]
LGTM 👍 

Thanks for spotting this and contributing an update. I'll apply these review suggestions myself :)

**Comment by polarathene** [comment]
## Before (v12.1)

image

## After (v13)

image

**Review by polarathene — APPROVED** [review]

**Comment by jpduyx** [comment]
thank you :-)

**Pull request merged**